### ISSUE TITLE
Warning about `SubfieldBase` deprecation

### ISSUE THREAD

**Issue by rowanv** [jjkester/django-auditlog#52]
I've installed the app and installation looks good, tests run fully and I am seeing expected functionality. However, when running all `mange.py` commands post-installation, I get the following deprecation warnings:

```
e:\Utilities\Python\Python34\lib\site-packages\django\utils\six.py:808: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
  return meta(name, bases, d)

e:\Utilities\Python\Python34\lib\site-packages\jsonfield\fields.py:139: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
  class TypedJSONField(JSONField):
```

Using django 1.9.4, django-auditlog 0.3.3.

**Comment by jjkester** [comment]
Thanks for your report. Apparently `SubFieldBase` will no longer be supported in Django 11 and upwards. Good news about that is that Django 11 will most likely be at least a year away, so I'll have plenty of time to fix this.

`TypedJSONField` is actually part of an external package Auditlog depends on. I'll check out this package and resubmit the issue there if it has not been submitted already. It could just be that I just need to update the version numbers...

Will be continued!

**Comment by rowanv** [comment]
Thanks for the prompt reply and the explanation @jjkester!

**Comment by jjkester** [comment]
The author of `django-jsonfield` fixed this in their 1.0.0 release, I will update the `requirements.txt` accordingly. This should get rid of the warnings.
